src/ray/object_manager/plasma/BUILD.bazel

```diff
@@ -0,0 +1,399 @@
+load("@com_github_google_flatbuffers//:build_defs.bzl", "flatbuffer_cc_library")
+load("//bazel:ray.bzl", "COPTS", "FLATC_ARGS", "ray_cc_library", "ray_cc_test")
+
+# TODO(mehrdadn): (How to) support dynamic linking?
+PROPAGATED_WINDOWS_DEFINES = ["ARROW_STATIC"]
```

Comment:
nit: starlark mostly follows python grammar, so prefer to use `_` prefix to indicate private variables.